web/endpoint.go

```diff
@@ -1,3 +1,24 @@
 // Package web provides managing HTTP protocol.
 // Implement CRUD operations for filters over HTTP API.
 package web
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/lvl484/positioning-filter/repository"
+)
+
+func newRouter(filters repository.Filters) *mux.Router {
+	handle := newRepo(filters)
+
+	router := mux.NewRouter()
+	filtersRouter := router.PathPrefix("/v1/router/{user_id}/filters").Subrouter()
```

Comment:
Please look into API specification, we have no such route

Reply:
fixed